.circleci/config.yml

```diff
@@ -68,7 +68,9 @@ jobs:
       - run: bin/install-chrome
       - run: bin/install-clojure
       - run: sudo bin/install-node && npm install
-      - run: lein check
+      - run:
+          name: 'Check artifact isolation + lack of reflection/boxing warnings'
+          command: lein with-profile -dev,+check check 2>&1 | grep warning && exit 1 || exit 0
```

Comment:
In case you are curious, this unix incantation has already been tried and tested in https://github.com/nedap/speced.def/pull/89